src/renderer/App.tsx

```
@@ -177,78 +160,58 @@ export default function App() {
+        toast.success("Screenshot added to input");
       } else if (result.success) {
-        toast.success(result.message || 'Screenshot completed');
+        toast.success(result.message || "Screenshot completed");
       } else {
         // Only show error toast if it's not a cancellation
-        if (result.error !== 'Screenshot was cancelled') {
+        if (result.error !== "Screenshot was cancelled") {
           toast.error(`Screenshot failed: ${result.error}`);
         }
       }
     } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)